Part 2 - Sequence Basics/1. Creating a sequence.md

```diff
@@ -246,7 +246,7 @@ Depending on what the event is, the event type (here `ActionEvent`) may be meani
 
 Much like most of the functions we've seen so far, you can turn any kind of input into an Rx observable with `create`. There are several shorthands for converting common types of input.
 
-`Future`s are part of the Java framework and you may come across them while using frameworks that use concurrency. They are a less powerful concept for concurrency than Rx, since they only return one value. Naturally, you'd like to  them into observables.
+`Future`s are part of the Java framework and you may come across them while using frameworks that use concurrency. They are a less powerful concept for concurrency than Rx, since they only return one value. Naturally, you'd like them to return into observables.
```

Comment:
"to return into observables" doesn't make much sense either.
"you'd 'like to turn them into observables" is probably what I meant to write.